src/python/CRABClient/ClientMapping.py

```diff
@@ -67,15 +67,17 @@
 }
 
 renamedParams = {
-    'General.transferOutput' : 'General.transferOutputs',
-    'General.saveLogs'       : 'General.transferLogs',
-    'Data.outlfn'            : 'Data.outLFN',
-    'Data.dbsUrl'            : 'Data.inputDBS',
-    'Data.publishDbsUrl'     : 'Data.publishDBS',
-    'Data.userInputFile'     : 'Data.userInputFiles',
-    'JobType.numcores'       : 'JobType.numCores',
-    'JobType.maxmemory'      : 'JobType.maxMemoryMB',
-    'JobType.maxjobruntime'  : 'JobType.maxJobRuntimeMin'
+    'General.transferOutput'          : 'General.transferOutputs',
+    'General.saveLogs'                : 'General.transferLogs',
+    'Data.outlfn'                     : 'Data.outLFN',
+    'Data.outLFN'                     : 'Data.outLFNDirBase'
+    'Data.dbsUrl'                     : 'Data.inputDBS',
+    'Data.publishDbsUrl'              : 'Data.publishDBS',
+    'Data.userInputFile'              : 'Data.userInputFiles',
+    'JobType.numcores'                : 'JobType.numCores',
+    'JobType.maxmemory'               : 'JobType.maxMemoryMB',
+    'JobType.maxjobruntime'           : 'JobType.maxJobRuntimeMin'
+    'JobType.allowNonProductionCMSSW' : 'JobType.allowUndistributedCMSSW'
```

Comment:
On the other hand, current guidelines generally say that you should avoid adding all these extra spaces just to get things to align nicely. Again, it's more difficult to see what you are actually changing here as opposed to just spacing out differently. And it's a waste of time.